src/lib.rs

```diff
@@ -13,12 +18,48 @@ use std::sync::{ Arc, Mutex, MutexGuard };
 use std::sync::mpsc;
 
 #[cfg(windows)]
-fn get_default_device() -> Result<&'static str> {
-    "\\\\.\\COM6"
+fn get_default_device() -> Result<String> {
+    "\\\\.\\COM6".to_owned()
+}
+
+fn is_usb_zwave_device(port: &ListPortInfo) -> bool {
+    let default_usb_devices = [
+        // VID     PID
+        //-----   -----
+        (0x0658, 0x0200),   // Aeotech Z-Stick Gen-5
+    ];
+
+    // Is it one of the vid/pids in the table?
+    if let UsbPort(ref info) = port.port_type {
+        default_usb_devices.iter().find(|entry| entry.0 == info.vid && entry.1 == info.pid).is_some()
+    } else {
+        false
+    }
 }
 
 #[cfg(unix)]
-fn get_default_device() -> Result<&'static str> {
+fn get_default_device() -> Result<String> {
+
+    // Enumerate all of the serial devices and see if any of them match our
+    // known VID:PID.
+
+    if let Some(port) = ListPorts::new().iter().find(|port| is_usb_zwave_device(port)) {
+        if let UsbPort(ref info) = port.port_type {
+            info!("Found ZWave USB dongle {:04x}:{:04x} on {}", info.vid, info.pid, port.device.display());
+        }
+        return Ok(port.device.to_string_lossy().into_owned());
+    }
+
+    // The following is only included temporarily until we can get a more
+    // comprehensive list of VIDs and PIDs.
+
+    error!("Unable to locate ZWave USB dongle. The following VID:PIDs were found:");
+    for port in ListPorts::new().iter() {
+        if let UsbPort(ref info) = port.port_type {
+            error!("  {:04x}:{:04x} {}", info.vid, info.pid, port.device.display());
+        }
+    }
```

Comment:
As the comment said, this is just temporary (perhaps for a couple weeks) until we figure out the VIDs/PIDs that everyone is using.

If we make if debug! then it just doesn't work and people have to know how to run it in debug mode in order to tell which VID:PID we should add to the table.